### PR TITLE
Update converter docs

### PR DESCRIPTION
**Pull request by ischoegl**
**Changes proposed in this pull request**

- fix links in converter script (`convert` and `main` are valid links)
- use Sphinx 'note' directives for improved browsing

**Checklist**

- [x] The pull request includes a clear description of this code change
- [x] Commit messages have short titles and reference relevant issues
- [x] Build passes (`scons build` & `scons test`) and unit tests address code coverage
- [x] Style & formatting of contributed code follows [contributing guidelines](https://github.com/Cantera/cantera/blob/main/CONTRIBUTING.md)
- [x] The pull request is ready for review

### CODE REVIEW

File: interfaces/cython/cantera/ck2yaml.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 """Convert Chemkin-format mechanism files to YAML.
 
-There are two main entry points to this script, ``main`` and ``convert``. The former is
```
Comment (bryanwweber) [inline interfaces/cython/cantera/ck2yaml.py:9]: Shouldn't these be reST formatted strings, needing two backticks to monospace?

Comment (ischoegl) [reply interfaces/cython/cantera/ck2yaml.py:9]: No. They are links to items within the document, see [current documentation](https://cantera.org/documentation/docs-3.0/sphinx/html/cython/cti_conversion.html). I got tripped up by a similar comment in the previous PR, but it was actually intentional 😄

**Review by speth — COMMENTED** [review]
Instead of the generic `note`, you can also use the `seealso` admonition, which is specifically styled by `pydata-sphinx-theme` (https://pydata-sphinx-theme.readthedocs.io/en/stable/examples/kitchen-sink/admonitions.html)

**Comment by ischoegl** [comment]
> Instead of the generic `note`, you can also use the `seealso` admonition, which is specifically styled by `pydata-sphinx-theme` (https://pydata-sphinx-theme.readthedocs.io/en/stable/examples/kitchen-sink/admonitions.html)

Great suggestion! Wasn't aware of this admonition, but it fits perfectly.

**Review by speth — APPROVED** [review]

**Pull request merged**